Buildings/Controls/OBC/FDE/PackagedRTUs/Validation/BASController_with_Requests.mo

```diff
@@ -0,0 +1,156 @@
+    annotation (Line(points={{-74,-14},{-70,
+          -14},{-70,0.8},{-66,0.8}}, color={255,0,255}));
+  connect(SpaceTGen2.y, SimPartCon2.ZoneT)
+    annotation (Line(points={{-72,-48},{-68,
+          -48},{-68,-58.4},{-64,-58.4}}, color={0,0,127}));
+  connect(OccGen3.y, SimPartCon2.OccOvrd)
+    annotation (Line(points={{-72,-80},{-68,
+          -80},{-68,-65.2},{-64,-65.2}}, color={255,0,255}));
+  connect(occReqTotal.y, basCont.occReq) annotation (Line(points={{36,58},{62,58},
+          {62,6.02},{72,6.02}}, color={255,127,0}));
+  connect(sbcReqTotal.y, basCont.sbcReq) annotation (Line(points={{36,26},{58,26},
+          {58,2.64},{72,2.64}}, color={255,127,0}));
+  connect(sbhReqTotal.y, basCont.sbhReq) annotation (Line(points={{36,-6},{58,-6},
+          {58,-1},{72,-1}}, color={255,127,0}));
+  connect(coolReqTotal.y, basCont.totCoolReqs) annotation (Line(points={{36,-38},
+          {62,-38},{62,-4.64},{72,-4.64}}, color={255,127,0}));
+  connect(SimPartCon.termOccReq, occReqTotal.u[1]) annotation (Line(points={{-41.8,
+          59.8},{-11.9,59.8},{-11.9,62.6667},{12,62.6667}}, color={255,127,0}));
+  connect(SimPartCon1.termOccReq, occReqTotal.u[2]) annotation (Line(points={{-41.8,
+          -2.2},{-41.8,57.9},{12,57.9},{12,58}}, color={255,127,0}));
+  connect(SimPartCon2.termOccReq, occReqTotal.u[3]) annotation (Line(points={{-39.8,
+          -68.2},{-39.8,54.9},{12,54.9},{12,53.3333}}, color={255,127,0}));
+  connect(SimPartCon.termSBC, sbcReqTotal.u[1]) annotation (Line(points={{-41.8,
+          69.4},{-14.9,69.4},{-14.9,30.6667},{12,30.6667}}, color={255,127,0}));
+  connect(SimPartCon1.termSBC, sbcReqTotal.u[2]) annotation (Line(points={{-41.8,
+          7.4},{-14.9,7.4},{-14.9,26},{12,26}}, color={255,127,0}));
+  connect(SimPartCon2.termSBC, sbcReqTotal.u[3]) annotation (Line(points={{-39.8,
+          -58.6},{-39.8,19.7},{12,19.7},{12,21.3333}}, color={255,127,0}));
+  connect(SimPartCon.termSBH, sbhReqTotal.u[1]) annotation (Line(points={{-41.8,
+          63},{-41.8,-1.5},{12,-1.5},{12,-1.33333}}, color={255,127,0}));
+  connect(SimPartCon1.termHeatReq, sbhReqTotal.u[2]) annotation (Line(points={{-41.8,
+          4},{-14,4},{-14,-6},{12,-6}}, color={255,127,0}));
+  connect(SimPartCon2.termSBH, sbhReqTotal.u[3]) annotation (Line(points={{-39.8,
+          -65},{-39.8,-13.5},{12,-13.5},{12,-10.6667}}, color={255,127,0}));
+  connect(SimPartCon.termCoolReq, coolReqTotal.u[1]) annotation (Line(points={{-41.8,
+          72.2},{-41.8,17.1},{12,17.1},{12,-33.3333}}, color={255,127,0}));
+  connect(SimPartCon1.termCoolReq, coolReqTotal.u[2]) annotation (Line(points={{
+          -41.8,10.2},{-14.9,10.2},{-14.9,-38},{12,-38}}, color={255,127,0}));
+  connect(SimPartCon2.termCoolReq, coolReqTotal.u[3]) annotation (Line(points={{-39.8,
+          -55.8},{-13.9,-55.8},{-13.9,-42.6667},{12,-42.6667}},       color={255,
+          127,0}));
+  annotation (Icon(coordinateSystem(preserveAspectRatio=false), graphics={
```

Comment:
add info section, and create script for running the unit test `Simulate and Plot`.